js/tests/redux/selectors/fallbackSelectors.test.js

```diff
@@ -0,0 +1,69 @@
+import { getDescriptionFallback, getTitleFallback, getImageFallback } from "../../../src/redux/selectors/fallbackSelectors";
+
+// This mimics parts of the yoast-seo/editor store.
+const testState = {
+	analysisData: {
+		snippet: {
+			title: "Hello World!",
+			description: "I describe something?",
+		},
+	},
+	settings: {
+		socialPreviews: {
+			siteWideImage: "side-wide.png",
+		},
+	},
+	snippetEditor: {
+		data: {
+			snippetPreviewImageURL: "featured.png",
+		},
+	},
+};
+
+describe( getTitleFallback, () => {
+	it( "returns the snippit title as a fallback", () => {
+		const actual = getTitleFallback( testState );
+
+		const expected = "Hello World!";
+
+		expect( actual ).toEqual( expected );
+	} );
+} );
+
+describe( getDescriptionFallback, () => {
+	it( "returns the snippit description as a fallback", () => {
+		const actual = getDescriptionFallback( testState );
+
+		const expected = "I describe something?";
+
+		expect( actual ).toEqual( expected );
+	} );
+} );
+
+describe( getImageFallback, () => {
+	it( "returns the site wide image if it's set", () => {
+		const actual = getImageFallback( testState );
+
+		const expected = "side-wide.png";
+
+		expect( actual ).toEqual( expected );
+	} );
+} );
+
+describe( getImageFallback, () => {
+	it( "returns the site wide image as a fallback", () => {
+		const state = {
+			 ...testState,
+			settings: {
+				socialPreviews: {
+					siteWideImage: "",
+				},
+			},
+		};
+		const actual = getImageFallback( state );
+
+		const expected = "featured.png";
+
+		expect( actual ).toEqual( expected );
+	} );
+} );
```

Comment:
You can put this in one `describe` statement. That makes it more obvious that you are testing different cases for the same function.

```suggestion
describe( getImageFallback, () => {
	it( "returns the site wide image if it's set", () => {
		const actual = getImageFallback( testState );

		const expected = "side-wide.png";

		expect( actual ).toEqual( expected );
	} );
	
	it( "returns the site wide image as a fallback", () => {
		const state = {
			 ...testState,
			settings: {
				socialPreviews: {
					siteWideImage: "",
				},
			},
		};
		const actual = getImageFallback( state );
	
		const expected = "featured.png";
	
		expect( actual ).toEqual( expected );
	} );
} );
```